### PR TITLE
Asssignmet 4 - Andrea cambios finales

### PR DESCRIPTION
**Pull request by AndreaMejiaCh**
tarea 6 y 7

### CODE REVIEW

File: Assignment4/andreamejia-22c063/task07_AndreaCambios.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+from types import prepare_class
+from rdflib.plugins.sparql import prepareQuery
+from rdflib.namespace import RDF, RDFS
+ns = Namespace("http://somewhere#")
+# TO DO
+# Visualize the results
+q1 =prepareQuery('''
+  SELECT * WHERE {
+      ?Subject rdfs:subClassOf ns:LivingThing.
+      Optional{?x rdfs:subClassOf ?Subject}
```
Comment (dgarijo) [inline Assignment4/andreamejia-22c063/task07_AndreaCambios.py:35]: This does not list the subclass hierarchy, just 2 levels. Use a property chain

File: Assignment4/andreamejia-22c063/task07_AndreaCambios.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+
+q2 = prepareQuery('''
+  SELECT * WHERE
+  {
+
+    {?Subject rdf:type ns:Person}
+    union
+    {
+      ?S rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/andreamejia-22c063/task07_AndreaCambios.py:58]: same issue as above

File: Assignment4/andreamejia-22c063/task07_AndreaCambios.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+for r in g.query(q4):
+   print(r.conoce)
+
+"""**Task 7.5: List the entities who know at least two other entities in the graph**"""
+
+q5 = prepareQuery('''
+  SELECT ?subject ?o WHERE
+  {
+    ?subject foaf:knows ?o.
```
Comment (dgarijo) [inline Assignment4/andreamejia-22c063/task07_AndreaCambios.py:118]: This is not doing what the exercise requests.
Also, 7.4 does not return name.
Also, all rdflib exercises are missing.

**Review by dgarijo — CHANGES_REQUESTED** [review]
Andrea, do not open a new PR. Work on this one. You are opening pull requests that contain the same errors, and we have to assess them over and over again. 

We will not assess new pull requests unless you do them here so we can see which changes you add

**Comment by anaigmo** [comment]
Entregado, corregido pero con fallos y faltan ejercicios, debería haber un archivo por ejercicio

**Pull request merged**